pkg/oci/utils.go

```diff
@@ -170,6 +170,13 @@ func PodConfig(runtime RuntimeConfig, bundlePath, cid, console string) (*vc.PodC
 		Console:     console,
 	}
 
+	if len(ocispec.Process.User.AdditionalGids) > 0 {
+		cmd.SupplementaryGroups = []string{}
```

Comment:
The difference is between passing nil vs an empty slice. I just preferred the earlier.